cmd/kube-proxy/app/server.go

```diff
@@ -108,6 +108,8 @@ type Options struct {
 	WriteConfigTo string
 	// CleanupAndExit, when true, makes the proxy server clean up iptables and ipvs rules, then exit.
 	CleanupAndExit bool
+	// InitAndExit, when true, makes the proxy server makes configurations that need privileged access, then exit.
```

Comment:
Nit: possibly typo here?

```diff
- InitAndExit, when true, makes the proxy server makes configurations that need privileged access, then exit.
+ InitAndExit, when true, makes the proxy server execute configurations that need privileged access, then exit.
```

Reply:
Lars meant "it makes the proxy server make changes to the node configuration that need privileges", whereas I think you're interpreting it as "it makes the proxy server process the kube-proxy configuration options that need privileges".

Maybe just copy the text from the CLI flag below: "InitAndExit, when true, makes kube-proxy perform any initialization steps that must be done with full root privileges, and then exit"

Reply:
I don't think it matters much. People that are interested in this function will get it anyway, so I let it be